src/components/EducationAchievements/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class EducationAchievements extends Component {
+  render() {
+    const { sampleEducation } = this.props;
+    console.log(sampleEducation);
+
+    return (
+      <ul className="achievements__list">
+        {sampleEducation.map((item, index) => {
+          return (
+            <li key={index} className="achievements__list-item">
+              {item.achievements}
```

Comment:
`item` would already be the exact achievement item. Right now, they appear in the DOM one just next to the other

Reply:
@lauraferrandof good catch! And sorry for the mixing up of array/object. I agree with changing the props being passed from the EducationItem to EducationAchievements, it solves the iteration issue. A nice and clean explanation, Laura!

Reply:
@emma-martin don't worry about the mixing up, I just pointed it out so that you don't have any problems with that right now :)
let's see what the others think!